nncf/onnx/graph/model_transformer.py

```diff
@@ -99,21 +101,25 @@ def transform(self, transformation_layout: TransformationLayout) -> onnx.ModelPr
                 quantizer_insert_transformations.append(transformation)
             elif isinstance(transformation, ONNXOutputInsertionCommand):
                 output_insert_transformations.append(transformation)
-            elif isinstance(transformation, ONNXBiasCorrectionCommand):
```

Comment:
What is the reason to move it?

Reply:
We should update the weight after removing Q-DQ. Aligned with OV backend.